socketdev/fullscans/__init__.py

```diff
@@ -0,0 +1,64 @@
+import socketdev
+from socketdev.tools import load_files
+
+class Fullscans:
+    @staticmethod
+    def get(org_slug: str) -> dict:
+        path = "orgs/"+org_slug+"/full-scans"
+        headers = None
+        payload = None
+
+        response = socketdev.do_request(
+            path=path,
+            headers=headers,
+            payload=payload
+        )
+        if response.status_code == 200:
+            result = response.json()
+        else:
+            result = {}
+        return result
+    
+    @staticmethod
+    def post(
+            files: list, 
+            params: dict
+            ) -> dict:
+        
+        loaded_files = []
+        loaded_files = load_files(files, loaded_files)
+
+        params_arg = "?"
+        params_arg  = params_arg +"repo="+params["repo"]
```

Comment:
This can probably be simplified to:

````
params_arg = ""
for name in params:
    value = params[name]
    params_arg += f"&{name}={value}"
params_arg = "?" + params_arg.lstrip("&")
````

This way in the future if there are new params they don't have be hard coded.

Reply:
Great feedback.  Thank you.

Reply:
Added how I think it should be done as a new commit.